yarpnarp.go

```diff
@@ -80,10 +104,26 @@ func RunYarpNarp() {
 
 	sort.Sort(zusagen)
 
+	var width int
+	var winsize struct {
+		wsRow    uint16
+		wsCol    uint16
+		wsXpixel uint16
+		wsYpixel uint16
+	}
+
+	if _, _, errno := unix.Syscall(unix.SYS_IOCTL, os.Stdout.Fd(), unix.TIOCGWINSZ, uintptr(unsafe.Pointer(&winsize))); errno != 0 {
```

Comment:
Das sollte denk ich ausgelagert werden in eine eigene Funktion -- und diese Funktion sollte dann ausgelagert werden in eine eigene Datei, damit wir [build constraints](https://golang.org/pkg/go/build/) nutzen können, sodass das nur unter linux gebaut wird (und einen stub für alles andere der einfach ne Konstante ausgibt für alle anderen Plattformen).

Es ist unwahrscheinlich dass das unter nicht-linux jemals laufen soll, aber zumindest wenn das dann bauen würde wäre das cooler :)